rustfmt.toml

```diff
@@ -13,7 +13,7 @@ ignore = [
 
     # tests for now are not formatted, as they are sometimes pretty-printing constrained
     # (and generally rustfmt can move around comments in UI-testing incompatible ways)
-    "/tests/",
```

Comment:
It's safe to remove this path since it's not whitelisted, nothing will be done on it apparently.

Reply:
What do you mean by whitelisted? Where is that list?

Reply:
The way we build the "walker" with the `ignore` crate is a bit strange. From what I understood, if something is not added to the list, it's ignored by default.

In this case for example, I removed the `tests` folder and added `rmake.rs` files and only the `rmake.rs` files have been formatted.